CHANGELOG.md

```diff
@@ -5,18 +5,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased](https://github.com/usgs/waterdataui/compare/waterdataui-0.37.0...master)
-<<<<<<< HEAD
 ### Added
-- A set of device specific favicons
```

Comment:
Looks like you have removed all of Aaron's changes to the CHANGELOG.md. You need to merge your changes with his.